src/Microsoft.VisualStudio.Services.Agent/Blob/DedupManifestArtifactClientFactory.cs

```diff
@@ -124,7 +124,7 @@ private DedupManifestArtifactClientFactory()
                     domainId,
                     clientSettings,
                     context,
-                    cancellationToken);            
+                    cancellationToken);
         }
 
         public (DedupManifestArtifactClient client, BlobStoreClientTelemetry telemetry) CreateDedupManifestClient(
```

Comment:
>CreateDedupManifestClient [](http://example.com/codeflow?start=88&length=25)

Ideally `CreateDedupManifestClient` and `CreateDedupClientAsync` would both call a common function that creates and initializes an `IDedupStoreHttpClient`

Reply:
will be in another PR